sw/device/tests/rv_timer_test.c

```diff
@@ -2,7 +2,7 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
```

Comment:
This is a sanity test, right?

Maybe just do the move right here in this PR?

Reply:
Done.